DESCRIPTION

```diff
@@ -46,10 +46,14 @@ RoxygenNote: 7.2.3
 Roxygen: list(markdown = TRUE)
 Authors@R: c(
     person("Gábor", "Csárdi", , "csardi.gabor@gmail.com", role = "aut"),
-    person("Tamás", "Nepusz", , "ntamas@gmail.com", role = c("aut", "cre"),
+    person("Tamás", "Nepusz", , "ntamas@gmail.com", role = "aut",
            comment = "See also AUTHORS file."),
     person("Vincent", "Traag", role = "aut"),
     person("Szabolcs", "Horvát", , "szhorvat@gmail.com", role = "aut"),
-    person("Fabio", "Zanini", , "fabio.zanini@fastmail.fm", role = "aut")
+    person("Fabio", "Zanini", , "fabio.zanini@fastmail.fm", role = "aut"),
+    person("Daniel", "Noom", role = "aut"),
+    person("Kirill", "Müller", role = c("aut", "cre")),
+    person("Maëlle", "Salmon", role = "ctb", email = "kirill@cynkra.com"),
```

Comment:
The email does not match the name here.

Reply:
oops, wrong copy paste, thanks a ton for catching!